automation-example/pages/Messages/Success.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
I'm not sure if this is good approach... this is not PAGE, just text message and why is this file not part of autoload or part of page where is located.... missing context.